tests/console/suseconnect_scc.pm

```diff
@@ -41,7 +41,7 @@ sub run {
     assert_script_run q[SUSEConnect --list-extensions | grep -e '\(Activated\)'] if is_sle;
 
     # add modules
-    register_addons_cmd if $scc_addons;
+    register_addons_cmd($scc_addons, 5) if $scc_addons;
```

Comment:
there is also retry in line 37 , maybe we can extract amount of retries to a variable ?

Reply:
yes, good point